src/SearchField/SearchField.scss

```diff
@@ -1,83 +1,21 @@
-$search-field-clear-btn-width: 30px;
-$search-field-border-radius: 30px;
-
-.search-field {
+.pgn__searchfield {
+    @extend .border;
+    @extend .rounded;
     transition: $input-transition;
-    border-radius: $search-field-border-radius;
-    box-shadow: $input-box-shadow;
-    background: #ffffff;
-
-    .form-group {
-        margin-bottom: 0;
-    }
-
-    .input-group {
-        flex: 1 1 0;
-
-        //clear browser 'x' from Internet Explorer
-        input[type=search]::-ms-clear {
-          display: none;
-          width : 0;
-          height: 0;
-        }
-        input[type=search]::-ms-reveal {
-          display: none;
-          width : 0;
-          height: 0;
-        }
-
-        //clear browser 'x' from Chrome
-        input[type="search"]::-webkit-search-decoration,
-        input[type="search"]::-webkit-search-cancel-button,
-        input[type="search"]::-webkit-search-results-button,
-        input[type="search"]::-webkit-search-results-decoration {
-          display: none;
-        }
-    }
-
-    label {
-        margin: 0;
-        padding-left: 20px;
-        padding-right: 10px;
-    }
-
-    button:last-child {
-        border-radius: $search-field-border-radius;
-    }
 
-    &.focused {
-        border-color: $blue !important;
+    &.has-focus {
         box-shadow: $input-focus-box-shadow;
     }
 
-    .input {
-        background: none;
-        border: 0;
-        box-shadow: none;
-        padding-left: 0;
-        padding-right: 0;
-        margin-top: 0.15rem;
+    .form-control {
+        @extend .border-0;
```

Comment:
border-0 will get you `!important` declarations I think. Might be better not to extend.

Reply:
Yes, extending does use `!important`. In this particular case, I think it makes sense since the border you would find on a normal `.form-control` is added to the `.pgn__searchfield` wrapper. If `!important` is not included on the `.form-control`, we are potentially making it easier for a consumer of `<SearchField />` to have a double border of the wrapper plus the `.form-control` if they were to override `border: 0;` (without the `!important` declaration).

Just my two cents. Happy to not use `@extend`, too, if we feel that's more appropriate.

Reply:
Should it be a form-control at all? Maybe like .form-control-file we should add a new class .form-control-search or something.  !important shouldn't be needed to customize stuff I think.